src/tests/sqlsmith/src/expr.rs

```diff
@@ -52,6 +59,17 @@ fn init_agg_table() -> HashMap<DataTypeName, Vec<AggFuncSig>> {
     funcs
 }
 
+fn init_cast_table() -> HashMap<DataTypeName, Vec<CastSig>> {
+    let mut casts = HashMap::<DataTypeName, Vec<CastSig>>::new();
+    cast_sigs()
+        .filter(|cast| cast.context == CastContext::Explicit) // TODO: generate implicit casts.
+        .filter(|cast| {
+            cast.from_type != DataTypeName::Varchar || cast.to_type == DataTypeName::Varchar
```

Comment:
Does it mean that we have some bugs on varchar or other reason? Pls add a line of comment

Reply:
> Avoids testing varchar casting, arbitrary string of characters may not be able to cast to datatypes